### PR TITLE
feat: add trackPadding and trackBg to Segmented tokens

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/pull/46688

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Segmented  add `trackPadding` and `trackBg` tokens.         |
| 🇨🇳 Chinese |    Segmented 新增 `trackPadding` and `trackBg` 两个 token。       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by yoyo837 — APPROVED** [review]

File: components/segmented/style/index.ts [hunk 1]
```diff
@@ -36,11 +36,19 @@ export interface ComponentToken {
    * @descEN Text color of item when selected
    */
   itemSelectedColor: string;
+  /**
+   * @desc Segmented 控件的 padding
+   * @descEN Padding of Segmented control
+   */
+  controlPadding: string | number;
```
[the remark this answers is not in the document]
Comment (zombieJ) [reply components/segmented/style/index.ts:43]: 是不是应该参考 Multiple Select 的 optionXXX 方式，反向算出来？

[the remark this answers is not in the document]
Comment (zombieJ) [reply components/segmented/style/index.ts:43]: cc @MadCcc

[the remark this answers is not in the document]
Comment (afc163) [reply components/segmented/style/index.ts:43]: 这个是 segmented 的变量，但是不能叫 itemXxx，好像没有更合适的名字了。

[the remark this answers is not in the document]
Comment (afc163) [reply components/segmented/style/index.ts:43]: 有没有名字推荐一个。

**Pull request merged**